_ramblings/2018-07-11-building-a-shell-part-1.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Building a Shell - Part 1
+layout: rambling
+draft: true
+---
+
+A shell is one of the best computer interfaces I've ever used.
+It's *declarative*, before being declarative was cool. It's
```

Comment:
Sequence of tenses issue

Reply:
Yea I struggled with this one. I had it as "It was *declarative*, before being declarative was cool." and it sounded off, but I agree this sounds more off now that I re-read it.